po/de.po

```diff
@@ -36,13 +36,11 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: GnuCash ${VERSION}-pot${POTVERSION}\n"
-"Report-Msgid-Bugs-To: https://bugs.gnucash.org/enter_bug.cgi?"
-"product=GnuCash&component=Translations\n"
+"Report-Msgid-Bugs-To: https://bugs.gnucash.org/enter_bug.cgi?product=GnuCash&component=Translations\n"
```

Comment:
Note: We use gettext's default line lengh (IIRC 72). But that is no problem: Weblate or a `msgcat -o po/${LL}.po po/${LL}.po` will fix it.